libraries/common/components/Link/index.jsx

```diff
@@ -45,6 +48,18 @@ class Link extends Component {
       return;
     }
 
+    /**
+     * For external pages we need to use the app command
+     * iOS deeplinks work out of the box on
+    * android though just opens it in the webview no matter
+     * what. Also our router removes trailing slashes which
+     * might be needed for external urls.
+     */
+    if (this.props.openExtern) {
```

Comment:
unfortunately that function also removes trailing slashes and opening google analytics without the trailing slash leads to an 404. I wanted to avoid any changes in there as it is very central, and you might want to open certain pages to open ALWAYS in in a external browser instead of going through our router.

Reply:
https://github.com/shopgate/pwa/blob/v6.12.0/libraries/common/subscriptions/router.js#L62

Reply:
but anyway, i've fixed it now. It wasn't working in the html widget before.